scipy/stats/_multivariate.py

```diff
@@ -659,10 +685,8 @@ def rvs(self, mean=None, cov=1, size=1, random_state=None):
 
         """
         dim, mean, cov = self._process_parameters(None, mean, cov)
-
-        random_state = self._get_random_state(random_state)
-        out = random_state.multivariate_normal(mean, cov, size)
-        return _squeeze_output(out)
+        cov_info = _PSD(cov, allow_singular=True)
```

Comment:
So the frozen distribution got faster, but how about the unfrozen distribution?  Does calling `_PSD` just to get `L` add significant overhead?